### DISCUSSION TITLE
Assimilation of in-situ soil moisture data

### DISCUSSION THREAD

**Discussion by tais958** [NASA-LIS/LISF#1125] [Q&A] [answered]
Hi, I wonder if LIS has an existing interface to assimilate in-situ (point) soil moisture data? I see it supports many soil moisture products but did not find anywhere describes what I mention here. Could you please comment if you have any. Thanks!

**Comment by LIS-navari** [comment, marked as answer]
Hi, 

To my knowledge, we do not have interfaces to support in-situ measurements. 
To find the description of each data set, please see the LIS documentation on GitHub  under the "Data assimilation set:"
(https://github.com/NASA-LIS/LISF/blob/master/lis/configs/lis.config.adoc)

**Reply by tais958** [reply]
OK, thanks for the reply!
